opencog/python/web/api/apiatomcollection.py

```diff
@@ -51,7 +51,7 @@ def __init__(self):
         self.reqparse.add_argument('limit', type=int, location='args')
 
         super(AtomCollectionAPI, self).__init__()
-        self.atomspace = opencog.cogserver.get_server_atomspace()
+        # self.atomspace = opencog.cogserver.get_server_atomspace()
 
```

Comment:
Why did you comment out this line? If it doesn't use the default cogserver atomspace then how will one be able to access the content in the default cogserver-atomspace?